db/models.py

```diff
@@ -50,3 +52,65 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to='User',
+                             on_delete=models.CASCADE,
+                             related_name="orders")
```

Comment:
You're using a string to refer to the User model in the ForeignKey field. While this is not wrong and can be useful in avoiding circular import issues, it's better to use the `settings.AUTH_USER_MODEL` to refer to the user model. This allows for better flexibility if you decide to change the user model in the future.